### ISSUE TITLE
stats: report waiting for indexing

### ISSUE THREAD

**Issue by bragov4ik** [blockscout/blockscout-rs#1123]
It would be a better UX to explicitly report that statistics is not available due to blockscout being indexed.
Maybe also report other statuses, like ongoing update 🤔

**Comment by bragov4ik** [comment]
some related thoughts:
- postpone start of cron update to be after initial update
- have different statuses:
  - waiting for its turn (is even needed?)
  - initial update
  - in the process of scheduled update (same ? maybe just update/ready?)
  - up to date

**Comment by bragov4ik** [comment]
this status can be used for better integration tests, as they now wait for fixed 5 seconds (which misbehaves sometimes, i.e. when logging is on).
so this will make the wait more efficient.

also maybe introduce some general status for all statistics, and separately from this idea - status for stats in other pages (hmm)

**Comment by bragov4ik** [comment]
also it would be cool to have integration tests initialize only once, not again for each test
